solutionAgent/agent.py

```diff
@@ -82,7 +82,7 @@ def master_agent(self,summary):
         Returns: solution to the logs"""
 
         answer =  self.agent_executor.invoke({"input":f"""You are a software developer who has to provide short summary of available solutions to issues of a software.
-                                     Use websearch tool available to make your answer. You can ask multiple questions from the webagent. Use websearch agent to search about information. You can use the summary provided. Also provide the sources of your solutions.
+                                     Use websearch tool available to make your answer. You can ask multiple questions from the webagent. Use "websearch" agent to search about information use Action: websearch. You can use the summary provided. Also provide the sources of your solutions.
```

Comment:
Do you have any output generated after this change

Reply:
I observed that sometimes there was an error during testing. So, I added this. I do not have any saved output but it worked similar as before.